src/iso15118/d20/state/dc_charge_parameter_discovery.cpp

```diff
@@ -42,6 +42,81 @@ template <> void convert(BPT_DC_ModeRes& out, const d20::DcTransferLimits& in) {
     }
 }
 
+bool handle_compatibility_check(const d20::DcTransferLimits& evse_dc_limits, const session::feedback::DcMaximumLimits ev_max_limits) {
+    // In IEC 61851-23-3 a compatibility check is required
+    // There are voltage, current and power values indexed with CPD and RATED.
+    // The CPD is the communicated value in DC_ChargeParameterDiscoveryRes and RATED the value coming from
+    // the EVerest environment (power supply, grid), here we assume that both are the same
+
+    bool compatiblity_flag = true;
+    const auto ev_max_limits_cpd = ev_max_limits;
```

Comment:
Why did you add another copy here? Why not using the argument `ev_max_limits` as const ref

Reply:
If you like to use variables for this, it's better to use const ref instead of copying each variable.